src/hooks/useSynth.tsx

```diff
@@ -23,17 +23,26 @@ export const useSynth = () => {
     setFreq(value);
   };
 
-  const newWave = (newType: ToneOscillatorType): void => {
+  const newWave = (newType: Tone.ToneOscillatorType): void => {
     selectedTrack.synth.type = newType;
   };
 
+  const rhythm = 8;
```

Comment:
This `rhythm` variable is a good opportunity to create another global state atom so that we can control it from elsewhere in the app.

Reply:
i converted it to a `frequencyAtom` in the store.ts file, but i'm having trouble pushing to this branch for some reason. maybe that can be something we can solve together

Reply:
nevermind, figured it out. just uninstalled everything and reinstalled